dataverse-persistence/src/main/resources/db/migration/V47__addTitleTranslationFields.sql

```diff
@@ -0,0 +1,213 @@
+UPDATE datasetfieldtype dsf SET displayorder = displayorder + 3 WHERE dsf.displayorder > 2;
+
+INSERT INTO datasetfieldtype (id, advancedsearchfieldtype, allowcontrolledvocabulary, allowmultiples, description,
+                              displayformat, displayoncreate, displayorder, facetable, fieldtype, name, required, title,
+                              uri, validationformat, watermark, metadatablock_id, parentdatasetfieldtype_id,
+                              inputrenderertype, inputrendereroptions)
+VALUES (266, false, false, true, 'Dataset title translated into another language', '',
```

Comment:
We can't use id in a reliable way right now. We must depend on auto generated value.